src/App.vue

```diff
@@ -1,5 +1,8 @@
 <template>
-  <v-app class="app-container" id="app">
+  <v-app
+    id="app"
+    class="app-container"
+  >
```

Comment:
The lint plugin automatically changed this to 1 attribute/prop per line, and moved `id` first.

Ditto below for `v-show` etc.